### DISCUSSION TITLE
Building upon first time contribution with GitHub org recognition feature

### DISCUSSION THREAD

**Discussion by 0-vortex** [open-sauced/open-sauced#1102] [Ideas]
Building upon the "identify your first open-source contribution" we could make a dashboard section or caching action script to get a list of all the different organizations the logged-in has contributed to. 

An example extension to data.json would look like this:

```json
{
    "contributions": [{
        "name": "open-sauced",
        "date": "2021-03-20"
    }],
    "stars": [{
            "full_name": "torvalds/linux",
            "stargazers_count": 115039,
            "open_issues_count": 324,
            "forks_count": 37971
        },
        {
            "full_name": "Team-TAU/tau",
            "stargazers_count": 45,
            "open_issues_count": 18,
            "forks_count": 17
        }
    ]
}
```

Additional information like logo, member count, etc might be useful depending on how we want to display this in the UI and whether we use persisted queries or cached data.

What do you think?

**Comment by E-cmyk-ctrl** [comment]
:loudly_crying_face(smiley):
